rectools/models/popular.py

```diff
@@ -87,25 +156,23 @@ class PopularModel(FixedColdRecoModelMixin, ModelBase):
     recommends_for_warm = False
     recommends_for_cold = True
 
+    config_class = PopularModelConfig
+
     def __init__(
         self,
         popularity: tp.Literal["n_users", "n_interactions", "mean_weight", "sum_weight"] = "n_users",
-        period: tp.Optional[timedelta] = None,
+        period: tp.Optional[TimeDelta] = None,
```

Comment:
You don't need annotated version here, it works only with `pydantic`

Reply:
Fixed.